### PR TITLE
feat: hide preview comment

### PR DESCRIPTION
**Pull request by OxrxL**
Immagine 2022-10-25 200007

https://github.com/revanced/revanced-integrations/pull/185

# TODO:

- [x] Remove the existing comments removal code from the `general-ads` patch to its own patch.

### CODE REVIEW

**Comment by oSumAtrIX** [comment]
Did you consider [this](https://github.com/revanced/revanced-patches/blob/f0d2f3e01b90bbe6d2b55b941eaf96be2295cd3c/src/main/kotlin/app/revanced/patches/youtube/ad/general/bytecode/patch/GeneralBytecodeAdsPatch.kt#L109)?

**Comment by OxrxL** [comment]
> Did you consider [this](https://github.com/revanced/revanced-patches/blob/f0d2f3e01b90bbe6d2b55b941eaf96be2295cd3c/src/main/kotlin/app/revanced/patches/youtube/ad/general/bytecode/patch/GeneralBytecodeAdsPatch.kt#L109)?

How this PR should give problems?

**Comment by OxrxL** [comment]
> Did you consider [this](https://github.com/revanced/revanced-patches/blob/f0d2f3e01b90bbe6d2b55b941eaf96be2295cd3c/src/main/kotlin/app/revanced/patches/youtube/ad/general/bytecode/patch/GeneralBytecodeAdsPatch.kt#L109)?

Nvm...my brain is out and I misunderstood again.

Anyway...this is not exactly an ad, so it's not worth to add it between ads.

**Review by oSumAtrIX — APPROVED** [review]

File: src/main/kotlin/app/revanced/patches/youtube/layout/comments/patch/CommentsPatch.kt [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import app.revanced.patches.youtube.layout.comments.annotations.CommentsCompatibility
+import app.revanced.patches.youtube.misc.mapping.patch.ResourceMappingResourcePatch
+import app.revanced.patches.youtube.misc.settings.bytecode.patch.SettingsPatch
+import app.revanced.patches.youtube.misc.settings.framework.components.impl.PreferenceScreen
+import app.revanced.patches.youtube.misc.settings.framework.components.impl.StringResource
+import app.revanced.patches.youtube.misc.settings.framework.components.impl.SwitchPreference
+
+@Patch
+@DependsOn([ResourceMappingResourcePatch::class, GeneralBytecodeAdsPatch::class])
+@Name("hide-comments-components")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/layout/comments/patch/CommentsPatch.kt:22]:
```suggestion
@Name("comments")
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Pull request merged**